drivers/flash/flash_stm32l4x.c

```diff
@@ -5,6 +5,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define SYS_LOG_DOMAIN "flash_stm32l4"
+#define SYS_LOG_LEVEL SYS_LOG_LEVEL_ERROR
+#include <logging/sys_log.h>
+
```

Comment:
this belongs into its own commit, has nothing to do with the rest of the change.

Reply:
moved to commit of `flash: stm32: stm32l4: add page layout support `